src/bindings/python/docs/test_examples.md

```diff
@@ -35,6 +45,31 @@ To run full test suite one can utilize `tox` command:
 tox
 ```
 
+### Check codestyle of Python API
```

Comment:
Shouldn't this be in `contributing.md`, in a single place?

@jiwaszki:
> I would like to keep codestyle in general contribution guide

Reply:
It also kind of testing API, also in `test_examples` I added info about running tests over API 1.0 which is missed in `contributind.md`